Mention dependencies on packaging page

I just noticed that hypothesis is broken in Macports because the `attrs` dependency is missing.

Apparently `attrs` was added as a required dependency in 3.28.0 - 2017-09-15 :
https://hypothesis.readthedocs.io/en/master/changes.html#id19

Here both attrs and coverage are listed as dependencies:
https://github.com/HypothesisWorks/hypothesis-python/blob/16a49c988a06d2c95bbee2e24e6638c92e81501d/setup.py#L59

Can you please update this page to explain the dependency on `attrs` and `coverage`?
https://hypothesis.readthedocs.io/en/master/packaging.html

I'll file an issue in the Macports tracker now asking them to add the attrs dependency.

Also, the error I'm getting is actually for `attr`, without the "s" in `attrs`!???
```
ImportError while importing test module '/private/var/folders/sb/4qv5j4m90pz1rw7m70rj1b1r0000gn/T/astropy_healpix-test-ubgaqiag/lib.macosx-10.13-x86_64-3.6/astropy_healpix/tests/test_healpy.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
astropy_healpix/tests/test_healpy.py:20: in <module>
    from hypothesis import given, settings
/opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/hypothesis/__init__.py:27: in <module>
    from hypothesis._settings import settings, Verbosity, Phase, HealthCheck, \
/opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/hypothesis/_settings.py:33: in <module>
    import attr
E   ModuleNotFoundError: No module named 'attr'
```
This is horribly confusing:
* https://pypi.python.org/pypi/attr
* https://pypi.python.org/pypi/attrs

--- a/docs/packaging.rst
+++ b/docs/packaging.rst
@@ -57,6 +57,7 @@ Hypothesis has *optional* dependencies on the following libraries:
 * :pypi:`faker`, version 0.7
 * `Django <https://www.djangoproject.com>`_, all supported versions
 * :pypi:`numpy`, 1.10 or later (earlier versions will probably work fine)
+* :pypi:`pandas`, 1.8 or later
 * :pypi:`py.test <pytest>` (2.8.0 or greater). This is a mandatory dependency for testing Hypothesis itself but optional for users.
 
 The way this works when installing Hypothesis normally is that these features become available if the relevant

--- a/docs/packaging.rst
+++ b/docs/packaging.rst
@@ -45,6 +45,11 @@ on either.
 Other Python libraries
 ~~~~~~~~~~~~~~~~~~~~~~
 
+Hypothesis has *mandatory* dependencies on the following libraries:
+
+* :pypi:`attrs`
+* :pypi:`coverage`
+
 Hypothesis has *optional* dependencies on the following libraries:
 
 * :pypi:`pytz` (almost any version should work)

--- a/docs/packaging.rst
+++ b/docs/packaging.rst
@@ -49,6 +49,7 @@ Hypothesis has *mandatory* dependencies on the following libraries:
 
 * :pypi:`attrs`
 * :pypi:`coverage`
+* :pypi:`enum34` is required on Python 2.7
 
 Hypothesis has *optional* dependencies on the following libraries:
 

